website/docs/discover.md

```diff
@@ -0,0 +1,319 @@
+to a specific registry (and, if you don't want to rely on a third-party registry, you don't to have to manage your own registry), you just
+need to have it stored in the container engine of the node of the Kubernetes cluster.
+
+You will be able to do this from the Images List page, in the Actions menu for an image. In this menu, an entry for each kind of cluster
+should be present, to push the image to a cluster of this kind.
+
+<figure>
+![Pushing a local image to a cluster](discover/img/move-image-to-node.png)
+<figcaption>Pushing a local image to a cluster</figcaption>
+</figure>
+
+#### Deploying a local container to a Kubernetes context
+
+Another very helpful tool is to "copy" a container running in a local container engine to a Kubernetes cluster, by creating
+a pod running this container in the cluster.
+
+To do this, you can choose a container in the Containers List page, and select the `Deploy to Kubernetes` action. This
+will show you the Kubernetes manifest that has been generated and which will be used to create the pod in the cluster,
+and will let you select a few options.
+
+After clicking the `Deploy` button, the Kubernetes manifest will be "applied" to the cluster,
+and you should be able to see the created pod in the Pods List page.
+
+<figure>
+![Deploying a container to a Kubernetes cluster](discover/img/deploy-container-to-kube.png)
+<figcaption>Deploying a container to a Kubernetes cluster</figcaption>
+</figure>
+
+#### Managing Kubernetes resources
+
+In the Pods List page, you will see both the pods managed by podman and the pods running in the current Kubernetes context,
+the Environment column displaying where each pod is running.
+
+By clicking the name of a Kubernetes pod, you will access the details of the pod, including the logs of the pod's containers
+streamed in real-time, a summary of the pod specification and status, and the possibility to access an interactive shell
+within the containers of the pod (for containers providing an `sh` or `bash` shell).
+
+In the Deployments, Services and Ingresses & Routes pages, you will see the corresponding resources existing in the
+current Kubernetes context.
+
+## Conclusion
+
+Podman Desktop is extensible. In fact, most of the features we have seen in this presentation are provided by extensions.
+
+An API is provided to write extensions in Javascript or Typescript. Extensions can register as a specific provider
+(authentication, registry, kubernetes, containers, cli tool, etc), can register to specific events (on images, containers, pods, etc),
+can add entries to menus (tray menu, status bar, contextual menus), and can contribute a complete UI through a webview provided by the API.
+
+Many extensions already exist, and many more are expected to come, which will make Podman Desktop a central place of choice for developers
+working with containers.
```

Comment:
maybe add item about the extension catalog

Reply:
I'm not sure what to share. Is there something else than `https://registry.podman-desktop.io/api/extensions.json`, human readable?

Reply:
@benoitf Do we have a repo which contains a sample extension? That would also be a great addition